pip/commands/install.py

```diff
@@ -213,6 +196,12 @@ def run(self, options, args):
             )
 
         if options.download_dir:
+            warnings.warn(
+                "pip install --download has been deprecated and will be "
+                "removed in the future. Pip now has a download command that "
+                "should be used instead.",
+                RemovedInPip8Warning,
```

Comment:
This should be RemovedInPip10Warning or maybe RemovedInPip9Warning ?
cc @dstufft, what is the deprecation policy ? 2 major versions ?